common/test/acceptance/pages/studio/index.py

```diff
@@ -28,18 +29,22 @@ def course_runs(self):
     def has_processing_courses(self):
         return self.q(css='.courses-processing').present
 
-    def create_rerun(self, display_name):
+    def create_rerun(self, course_key):
         """
-        Clicks the create rerun link of the course specified by display_name.
+        Clicks the create rerun link of the course specified by course_key
         """
-        name = self.q(css='.course-title').filter(lambda el: el.text == display_name)[0]
-        name.find_elements_by_xpath('../..')[0].find_elements_by_class_name('rerun-button')[0].click()
+        actions = ActionChains(self.browser)
+        button_name = self.browser.find_element_by_css_selector('.rerun-button[href$="' + course_key + '"]')
+        actions.move_to_element(button_name)
+        actions.click(button_name)
+        actions.perform()
 
     def click_course_run(self, run):
         """
         Clicks on the course with run given by run.
         """
         self.q(css='.course-run .value').filter(lambda el: el.text == run)[0].click()
+        self.wait_for_ajax()
```

Comment:
@raeeschachar In pull requests where you intentionally are adding in wait_for_ajax commands could you please add a comment there that says something like "Clicking on the course will trigger an ajax event"? This will help in future debugging, and will be a good example for other developers to follow. And that way we'll know that it wasn't just sprinkled in by someone who wasn't sure if it needed it or not but figured it wouldn't hurt.